src/main/java/de/zalando/ep/zalenium/container/kubernetes/KubernetesContainerClient.java

```diff
@@ -102,6 +103,7 @@ public KubernetesContainerClient(Environment environment,
             discoverHostAliases();
             discoverNodeSelector();
             discoverTolerations();
+            discoverPullSecrets();
```

Comment:
I think it's better to keep the full name as well. Please rename to `discoverImagePullSecrets()`